src/main/java/org/ibp/api/rest/germplasm/GermplasmResource.java

```diff
@@ -182,4 +183,16 @@ public long getFilteredCount() {
 		return new ResponseEntity<>(this.germplasmService.importGermplasm(cropName, programUUID, germplasmImportRequestDto), HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Import germplasm updates")
+	@RequestMapping(value = "/crops/{cropName}/germplasm", method = RequestMethod.PATCH)
+	@ResponseBody
+	public ResponseEntity<Void> importGermplasmUpdates(@PathVariable final String cropName,
+		@RequestParam(required = false) final String programUUID,
+		@RequestBody final List<GermplasmUpdateDTO> germplasmUpdateDTOList) {
```

Comment:
I suggest to rename this germplasmUpdateDTOList to germplasmList (or maybe there is anything in Swagger that allows us to show a different variable name) Not very important though